lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -4580,7 +4640,7 @@ void ByteCodeGenerator::EmitPropStore(Js::RegSlot rhsLocation, Symbol *sym, Iden
                                   slot + (sym->GetScope()->GetIsObject()? 0 : Js::ScopeSlots::FirstSlotIndex));
         }
         else if (scopeLocation != Js::Constants::NoRegister &&
-                 (scopeLocation == funcInfo->frameSlotsRegister || scopeLocation == funcInfo->frameObjRegister))
+            (scopeLocation == funcInfo->frameSlotsRegister || scopeLocation == funcInfo->frameObjRegister))
```

Comment:
Likewise I'm cautious here about needing to add this check to EmitPropStore. What case wasn't this catching? You might want to check with @pleath if you haven't already.